artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/ClientSessionFactoryImpl.java

```diff
@@ -238,6 +238,8 @@ public ClientSessionFactoryImpl(final ServerLocatorInternal serverLocator,
       if (connectorConfig.getB() != null) {
          this.backupConnectorConfig = connectorConfig.getB();
       }
+
```

Comment:
@brusdev I will send a PR removing this right now :)


that way you rebase without this...

Reply:
here https://github.com/apache/activemq-artemis/pull/3872

Reply:
great thanks